extra/emqx_base/README.md

```diff
@@ -0,0 +1,7 @@
+## EMQX Base Station
+
+Clone repo and cd to this directory.  
```

Comment:
What is this for? Aren't we just going to deploy using the web deployment interface?

Reply:
I didn't use the web interface simply bc then I knew exactly what command line parameters were being included.  I can try this out with the web interface it is just a matter of copying the various settings over.  However we still need to clone the repo.